src/Parallel/distance.jl

```diff
@@ -1,20 +1,51 @@
 # used in shortest path calculations
 
 function eccentricity(
+    g::AbstractGraph,
+    vs=vertices(g),
+    distmx::AbstractMatrix{T}=weights(g);
+    parallel=:distributed,
```

Comment:
```suggestion
    parallel::Symbol=:distributed,
```

Reply:
I know we haven't done that in other places - but perhaps to make this code more future proof would it make sense to use a specific structure to specify the parallel policy? Then we could later add things such as the number of threads/cores used.